src/v3/src/components/Images/AppIcon.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { FunctionComponent, h } from 'preact';
+
+import { loc } from '../../util/locUtil';
+
+export const AppIcon: FunctionComponent = () => (
+  <svg
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    role="img"
+    aria-labelledby="application-icon"
```

Comment:
Is it contradictory to have `aria-labelledby` _and_ `aria-hidden`?

Reply:
+1 

I would also say, having aria-labelledby is not necessary here, you can just use the title element on its own to pass a11y checks. 

We used it in other SVGs because the ID of the title was used in the aria-describedby of a different element (i.e. authenticator button). It doesn't seem like these are needed here.

Reply:
yeah since its hidden from screen reader it shouldn't be necessary